# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp
# Commit: 91d8f0f829fd
# Full Hash: 91d8f0f829fd2ae6ec234a6a3f372bf503ed2fbd
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 91cb8ee8ddd8 -r 91d8f0f829fd gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp
--- a/gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp	Thu Nov 05 19:26:21 2020 +0000
+++ b/gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp	Thu Nov 05 20:18:13 2020 +0000
@@ -34,7 +34,8 @@
   return gfx::ColorDepth::COLOR_8;
 }
 
-bool RenderSharedSurfaceTextureHostSWGL::MapPlane(uint8_t aChannelIndex,
+bool RenderSharedSurfaceTextureHostSWGL::MapPlane(RenderCompositor* aCompositor,
+                                                  uint8_t aChannelIndex,
                                                   PlaneInfo& aPlaneInfo) {
   if (NS_WARN_IF(
           !mSurface->Map(gfx::DataSourceSurface::MapType::READ, &mMap))) {